bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -78,6 +78,9 @@ public WebViewEnvironment(ICoreWebView2Environment environment) {
 	private boolean ignoreFocus;
 	private String lastCustomText;
 
+	record CursorPosition(Point location, boolean isInsideBrowser) {};
```

Comment:
I'd make it private

Reply:
Thanks! Yes, this should be private. Fixed with a76d981.